core/src/test/scala/module/sql/SqlQueryRunnerSpec.scala

```diff
@@ -29,7 +29,7 @@ class SqlQueryRunnerSpec(implicit ee: ExecutionEnv) extends Specification {
       val withSqlConnection: WithSqlConnection =
         SqlConnectionFactory.withSqlConnection(AcolyteQueryResult.Nil)
       val runner = SqlQueryRunner(withSqlConnection)
-      val query = SqlQuery.pure(1)
+      val query  = SqlQuery.pure(1)
```

Comment:
tjrs pas ça ...

Reply:
autant dans une for-comprenhension je peux comprendre, mais aligner dans ces cas me parait très bizarre